pkg/autoscaler/revision_scaler_test.go

```diff
@@ -146,6 +129,6 @@ func checkReplicas(t *testing.T, kubeClient kubernetes.Interface, deployment *v1
 	}
 
 	if *updatedDeployment.Spec.Replicas != int32(expectedScale) {
-		t.Fatal("Unexpected deployment replicas.", updatedDeployment.Spec.Replicas)
+		t.Fatal("Unexpected deployment replicas.", *updatedDeployment.Spec.Replicas)
```

Comment:
`Fatalf`?

Reply:
Do you mean, use `Fatalf` to print a "want x, got y" style message, or use `Fatalf` to not have to dereference the pointer?

Reply:
I just meant that seems more natural when passing args.